sktime/datasets/data/PBS_dataset/PBS_dataset

```diff
@@ -0,0 +1,205 @@
+Month,Scripts
```

Comment:
Please remove this file (it's the same as the CSV file)

Reply:
Done!